models/public/robust-video-matting/README.md

```diff
@@ -0,0 +1,226 @@
+# robust-video-matting
+
+## Use Case and High-Level Description
+
+The `robust-video-matting` model is a robust high-resolution human video matting method that
+achieves new state-of-the-art performance that uses a recurrent architecture to exploit temporal
+information in videos and achieves significant improvements in temporal coherence and matting quality.
+This model is pre-trained in PyTorch\* framework and converted to ONNX\* format. More details
+provided in the [paper](https://arxiv.org/abs/2108.11515). Backbone is MobileNetV3.
+For details see the [repository](https://github.com/DmitriySidnev/RobustVideoMatting).
```

Comment:
if I right understand you use fork for providing exporting code?
Maybe better to provide original repo as reference and additionally mention your scripts for pytorch to onnx conversion?

Reply:
Fixed.